pycbc/waveform/bank.py

```diff
@@ -68,6 +68,11 @@ class BaseFilterBank(object):
     about elements of an xml template bank.
     """
     def __init__(self, filename, approximant=None, **kwds):
+
```

Comment:
I don't understand why the following is required. Why don't the command line options to pycbc_inspiral contain the correct path (either relative or absolute) already?